src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,319 @@ paths:
         500:
           description: "Internal Server Error"
 
+  /alert-subscriptions:
+    post:
+      operationId: "create_and_subscribe_to_alert"
+      tags:
+        - "Alert subscriptions"
+      summary: Create an alert and subscribe users
+      description: This request associates a query to a newly created alert using an assetId property, and subscribes users to alerts for that query through the use of emailIds.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-service-token'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
+
+        - name: alert subscription
+          in: body
+          description: Create an alert for a particular query or schedule ID and subscribe emailIds to different notification channels.
+          required: true
+          schema:
+               $ref: '#/definitions/subscribe_user_to_alert_body'
+
+      responses:
+        200:
+          description: The subscription to an alert was successful.
+          schema:
+            $ref: '#/definitions/alert_subscription_response_body'
+        400:
+          description: "Bad Request. The payload may be malformed, the alert might be in a disabled state, or an emailID provided for subscribing to the alert might not be a registered Adobe ID."
+          schema:
+            $ref: '#/definitions/post_bad_request_error_message'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "Not found. The request cannot be processed because the asset ID provided is invalid."
+          schema:
+            $ref: '#/definitions/post_not_found_error_message'
+        500:
+          description: "Internal Server Error"
+    get:
+      operationId: "get_all_alerts_per_imsOrg_and_sandbox"
+      tags:
+        - "Alert subscriptions"
+      summary: Retrieve all alerts present in an organization sandbox
+      description: This request retrieves a list of all the alerts for an organization sandbox.
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
+
+      responses:
+        200:
+          description: All the alerts for an organization and a particular sandbox.
+          schema:
+            $ref: '#/definitions/alerts_per_imsorg_and_sandbox'
+        400:
+          description: "Bad Request. The emailID is not an Adobe registered ID."
```

Comment:
From Chhavi:
Line 847 848 are not required [ this API would not give 400]. I had it earlier in my swagger doc. but more recently i updated it.